wolfCLU/clu_include/clu_header_main.h

```diff
@@ -71,6 +72,12 @@
 #define MEGABYTE (1024*1024)
 #define MAX_THREADS 64
 #define MAX_FILENAME_SZ 256
+#define CLU_4K_TYPE 4096
+#if LIBWOLFSSL_VERSION_HEX >= 50413568 /* int val of hex 0x0301400 = 50413568 */
```

Comment:
This looks good but the macro in clu_include/version.h needs to be changed from LIBWOLFSSL_VERSION_HEX to some wolfCLU unique macro so that they do not clash.

Reply:
Good catch! Thanks!